### DISCUSSION TITLE
How to hide lualine by default and? How to use laststatus=3 by default instead of laststatus=2?

### DISCUSSION THREAD

**Discussion by uwla** [nvim-lualine/lualine.nvim#1292] [Q&A]
Hi, how can I hide lualine by default ? My vimrc sets `laststatus=0` but after `require('lualine').setup({})` it sets `laststatus=2`. I have a keymap that toggles laststatus and it starts hidden.

Also, when I need lualine to be visible I want  `laststatus=3` and not `laststatus=2`. How can I tell lualine to use `laststatus=3` instead of `2` such that I do not have to manually type it in the cmdline?

**Comment by iton0** [comment]
Check out this [section](https://github.com/nvim-lualine/lualine.nvim?tab=readme-ov-file#disabling-lualine) of the docs. Example config:
```lua
  'nvim-lualine/lualine.nvim',
  config = function()
    require('lualine').setup({})
    require('lualine').hide() -- hide on plugin loading
    vim.o.laststatus = 2 -- whatever you want your default laststatus to be

    -- Keymap to toggle `laststatus`
    vim.keymap.set('n', '<leader>t', function()
      if vim.o.laststatus == 2 then -- This should match whatever you want as your default laststatus that you put in your config above
        vim.o.laststatus = 3
        require('lualine').hide({ unhide = true }) -- Unhide lualine
      else
        require('lualine').hide() -- hide lualine
        vim.o.laststatus = 2
      end
    end, { noremap = true, silent = true })
  end,
```
This example assumes you are using lazy.nvim as your plugin manager.
